File: src/integTest/project/app/src/main/scala/de/ferey/android/hello/HelloActivity.scala
package de.ferey.android.hello

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class HelloActivity extends Activity {
  override def onCreate(savedInstanceState: Bundle) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_hello)
    val mirahTextView = findViewById(R.id.mirah_text_view).asInstanceOf[TextView]
    mirahTextView.setText(new HelloJava().say + "\n" + new HelloMirah().say)
  }
}

File: src/integTest/project/apt/src/main/java/de/ferey/android/hello/HelloActivity.java
package de.ferey.android.hello;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

import org.androidannotations.annotations.AfterViews;
import org.androidannotations.annotations.EActivity;
import org.androidannotations.annotations.ViewById;

@EActivity(R.layout.activity_hello)
public class HelloActivity extends Activity {
    @ViewById
    TextView mirahTextView;

    @AfterViews
    public void fillMirahTextView() {
        mirahTextView.setText(new HelloJava().say() /*+ "\n" + new HelloMirah().say()*/); // Joint compilation is currently not supported
    }
}
